src/uct/base/uct_iface.h

```diff
@@ -130,6 +130,17 @@ typedef struct uct_am_handler {
 
 
 /**
+ * Wakeup event handle item
+ */
+typedef struct uct_wakeup {
+    enum uct_wakeup_event_types events;
+    uct_iface_t                 *iface;
```

Comment:
pls make this the first field (and use uct_iface_h), so the memory layout will be more compact